### PR TITLE
Jenkins Automation: Add Common Publishers in Build Flow Template

### PR DESCRIPTION
**Pull request by amymok**
This branch is originated from the master branch of https://github.com/ooblioob/jenkins-automation
### Changes:
- Added 3 publishers items to build flow template:
  - aggregateBuildFlowTests
  - allowBrokenBuildClaiming()
  - extendedEmail()
### Reviews:

@Ooblioob

### CODE REVIEW

**Comment by Scotchester** [comment]
@Ooblioob Is this good to go?

File: jenkins-automation.groovy [hunk 1]
```diff
@@ -140,7 +211,7 @@ job('dsl-project-builder') {
   multiscm {
     git {
       remote {
-        url('https://github.com/cfpb/jenkins-automation')
+        url('https://github.com/Ooblioob/jenkins-automation')
```
Comment (Ooblioob) [inline jenkins-automation.groovy:214]: I'll submit a PR to set these back.

File: jenkins-bootstrap.groovy [hunk 2]
```diff
@@ -4,7 +4,7 @@ job('dsl-master') {
   multiscm {
     git {
       remote {
-        url('https://github.com/cfpb/jenkins-automation')
+        url('https://github.com/Ooblioob/jenkins-automation')
```
Comment (Ooblioob) [inline jenkins-bootstrap.groovy:7]: Same here

**Comment by Ooblioob** [comment]
Ok, we have some cleanup work to do at a later point, since many of the files here are just experiments to see what was possible.  The core of Amy's changes seems fine to me.  :+1:

**Pull request merged**